ses_accounts_mail_alias.tf

```
@@ -0,0 +1,21 @@
+module "ses-root-accounts-mail-alias" {
+  count     = var.ses_root_accounts_mail_alias != null ? 1 : 0
+  providers = { aws = aws, aws.route53 = aws }
+
+  source     = "github.com/schubergphilis/terraform-aws-mcaf-ses?ref=v0.1.0"
+  domain     = var.ses_root_accounts_mail_alias.domain
+  kms_key_id = module.kms_key.id
+  tags       = var.tags
+}
+
+module "ses-forwarder-root-accounts-mail-alias" {
+  count     = var.ses_root_accounts_mail_alias != null ? 1 : 0
+  providers = { aws = aws, aws.lambda = aws }
+
+  source            = "github.com/schubergphilis/terraform-aws-mcaf-ses-forwarder?ref=v0.1.1"
```

```suggestion
  count     = var.ses_root_accounts_mail_alias != null ? 1 : 0
  providers = { aws = aws, aws.lambda = aws }
  source    = "github.com/schubergphilis/terraform-aws-mcaf-ses-forwarder?ref=v0.1.1"

```

According to [the docs](https://www.terraform.io/language/modules/syntax#calling-a-child-module), source counts .as a meta-argument and thus should be moved to the top block